.pre-commit-config.yaml

```diff
@@ -10,8 +10,16 @@ repos:
     -   id: trailing-whitespace
 
 -   repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.3.5
+    rev: v0.4.8
```

Comment:
we can eliminate the need of manually adding the next version with the following lines

```yaml
ci:
  autofix_commit_msg: 🎨 [pre-commit.ci] Auto format from pre-commit.c
  autoupdate_commit_msg: ⬆ [pre-commit.ci] pre-commit au
  ```
  emoji can be omitted if you wish